ocrs/src/recognition.rs

```diff
@@ -500,7 +508,8 @@ impl TextRecognizer {
                             group_width as usize,
                         );
 
-                        let rec_output = self.run(rec_input)?;
+                        // Here mutation is added to allow whitelisting of characters
```

Comment:
I don't usually comment the reason for a value being mutable, because the compiler will tell you if `mut` is used unnecessarily.

Reply:
done. removed it